proto/vts.proto

```diff
@@ -38,7 +38,10 @@ enum EndorsementType {
 message Endorsement {
     AttestationFormat scheme = 1;
     EndorsementType type = 2;
-    google.protobuf.Struct attributes = 3;
+    /* sub_type is used by schemes to classify range of 
+    Endorsement sub types for a given Endorsement type */
```

Comment:
this is not the comment I meant.  What needs to be communicated is:

1. the fact that it is "public" but effectively opaque to Veraison
2. the fact that it assumes there is only ever going to be one single level of sub-type nesting

Reply:
For 1. We are saying that sub_type is used by schemes, is in effect saying that it is opaque to Veraison. Saying it is Opaque to Veraison will make it more  confusing as reader will not understand who is the user of this field. 

However, to make it more clear I will add is opaque to Veraison and used by schemes ....

2. This assumption is implicit as only one sub type field is their right? so can only have one usage, it is not an array nor this is an extensible type? Not sure what value add is achieved, by explicitly stating that ? However if you feel  it adds more clarity about it, I am happy to add this in comment

Reply:
> For 1. We are saying that sub_type is used by schemes, is in effect saying that it is opaque to Veraison. Saying it is Opaque to Veraison will make it more confusing as reader will not understand who is the user of this field.

I don't see the confusion.  If it is opaque to Veraison, Veraison is not the user. 

> However, to make it more clear I will add is opaque to Veraison and used by schemes

So is it "more clear" or "more confusing"?  Now *I* am confused :-)

> 2. This assumption is implicit as only one sub type field is their right? so can only have one usage, it is not an array nor this is an extensible type? Not sure what value add is achieved, by explicitly stating that ? However if you feel  it adds more clarity about it, I am happy to add this in comment

My point is exactly that the implicit assumption needs to be made explicit.

Reply:
Done!